src/main/java/com/limechain/runtime/hostapi/HostApi.java

```diff
@@ -74,6 +76,10 @@ protected static Memory getMemory() {
         return runtime.getInstance().exports.getMemory("memory");
     }
 
+    protected static int getHeapBase() {
+       return runtime.getInstance().exports.getGlobal("heap_base").getIntValue();
```

Comment:
The probably won't work as the heap base global is preceded by 2 underscores.
Also you can create an enum with the commonly exported WASM globals, `__heap_base` and `__data_end`. You can probably also add `memory` there as that would most likely always be the name of the exported/imported memory.